src/com/rcv/CVRReader.java

```diff
@@ -73,7 +73,7 @@ public void parseCVRFile(
     org.apache.poi.ss.usermodel.Row headerRow = iterator.next();
     // require at least one row
     if (headerRow == null || contestSheet.getLastRowNum() < 2) {
-      Logger.log("invalid RCV format: not enough rows:%d", contestSheet.getLastRowNum());
+      Logger.severe("invalid RCV format: not enough rows:%d", contestSheet.getLastRowNum());
```

Comment:
I strongly recommend making a general pass in all the code for instances of ":%" and replacing them with ": %". It will make the output much more readable.

Reply:
moved this comment to issue #74